### ISSUE TITLE
add option to search to return list of files suitable for rsyncing

### ISSUE THREAD

**Issue by dannyjacobs** [HERA-Team/librarian#18]
in addition to an html and text list

**Comment by pkgw** [comment]
As in, prefixed with the store host name and a colon?

**Comment by dannyjacobs** [comment]
not exactly sure whats best. This was requested by Jonnie in Cape Cod. Something that made it easy to download.

**Comment by pkgw** [comment]
CC @jpober ? Anything specific you remember having in mind? I can make it generate a script that you can run that will rsync everything.

**Comment by jpober** [comment]
That's basically it.  I was imaging that if I used the librarian to identify a list of files meeting a certain set of criteria, that it could then give me a list of absolute paths that I could use for rsync.  Whether it just gives me something to copy/paste onto the command line or a shell script that runs rsync, I suppose either would work, although the latter may be better if there are a lot of files.

**Comment by pkgw** [comment]
FWIW the "text file" output option will just give you a list of absolute paths right now, but the option to generate a script should be easy to implement and will probably make people's lives easier.

**Comment by dannyjacobs** [comment]
The one problem is that not all stores will be rsyncable from the outside world. Maybe a temporary tunnel?  I think this is one reason NGAS did http downloads.

**Comment by jpober** [comment]
Yeah, I guess I'm using rsync as a verb to mean transfer files.  Anything that will make it easy to locally download data sets found using the librarian search should get the job done.
